### PR TITLE
added styleci

### PR DESCRIPTION
**Pull request by lsmith77**

### CODE REVIEW

File: tests/Transport/Fs/Filesystem/Adapter/AdapterTestCase.php [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace Jackalope\Tests\Transport\Fs\Filesystem\Adapter;
+namespace Jackalope\tests\Transport\Fs\Filesystem\Adapter;
```
Comment (lsmith77) [inline tests/Transport/Fs/Filesystem/Adapter/AdapterTestCase.php]: the ``tests`` directory should I guess be renamed to ``Tests`` ?

Comment (dbu) [inline tests/Transport/Fs/Filesystem/Adapter/AdapterTestCase.php]: no. tests is the PSR-4 root for Jackalope\Tests . tests and src are kept separate and the root folder is not a code folder. i think its what most libraries do. usually only symfony bundles start with classes directly in the root directory.

Comment (lsmith77) [inline tests/Transport/Fs/Filesystem/Adapter/AdapterTestCase.php]: so this namespace renaming is an issue in the fixer?

Comment (dbu) [inline tests/Transport/Fs/Filesystem/Adapter/AdapterTestCase.php]: imo yes

Comment (lsmith77) [inline tests/Transport/Fs/Filesystem/Adapter/AdapterTestCase.php]: ping @GrahamCampbell

Comment (GrahamCampbell) [inline tests/Transport/Fs/Filesystem/Adapter/AdapterTestCase.php]: You need to disable the psr0 fixer.

**Comment by lsmith77** [comment]
@dbu ready to merge

**Comment by GrahamCampbell** [comment]
Just note that StyleCI is going down for essential maintenance in a few hours.

**Comment by GrahamCampbell** [comment]
We perform maintenance on Sunday afternoons if required, for minimal disturbance.

**Comment by lsmith77** [comment]
@dantleech not sure about the test failures ..

**Comment by GrahamCampbell** [comment]
Probably due to the broken namespace.

**Comment by GrahamCampbell** [comment]
You should revert the changes there.

**Comment by lsmith77** [comment]
doh of course .. keep forgetting that styleci doesn't magically apply the changes but that I apply them manually ;)

File: .styleci.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+preset: psr2
+
+enabled:
+  - long_array_syntax
+  - duplicate_semicolon
+
+disabled:
+  - psr0-fixer
```
Comment (lsmith77) [inline .styleci.yml]: is this the correct syntax?

Comment (GrahamCampbell) [inline .styleci.yml]: No

Comment (GrahamCampbell) [inline .styleci.yml]: `psr0`

**Comment by lsmith77** [comment]
build failures seem to be happening in master as well https://travis-ci.org/jackalope/jackalope-fs/builds/60118565

**Comment by dbu** [comment]
seems some of the newer bugfixes are missing here, and some issue with binary properties.

**Pull request merged**